spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/JdbcSearchableJobExecutionDao.java

```diff
@@ -107,8 +130,13 @@ public class JdbcSearchableJobExecutionDao extends JdbcJobExecutionDao implement
 	private PagingQueryProvider byTaskExecutionIdWithStepCountPagingQueryProvider;
 
 	private DataSource dataSource;
+	private final BatchVersion batchVersion;
 
-	/**
+    public JdbcSearchableJobExecutionDao(BatchVersion batchVersion) {

```

Comment:
If this class already exists we need to create the default constructor as this is a breaking change (if someone happens to be using this).

Reply:
Restored default constructor.